### PR TITLE
Speed up QR code scanner for iOS

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

**Comment by cmgustavo** [comment]
Android doesn't work with two camera related plugins: ```Error:
	Element uses-feature#android.hardware.camera at AndroidManifest.xml:44:5 duplicated with element declared at AndroidManifest.xml:43:5```

PR is still in progress.

**Comment by cmgustavo** [comment]
Updated

**Comment by matiu** [comment]
reviewing

File: cordova/build.sh [hunk 1]
```diff
@@ -84,8 +84,14 @@ if [ ! -d $PROJECT ]; then
   cordova plugin add https://github.com/florentvaldelievre/virtualartifacts-webIntent.git
   checkOK
 
-  cordova plugin add https://github.com/phonegap/phonegap-plugin-barcodescanner.git
-  checkOK
+  if [ $CURRENT_OS != "WP8" ]
```
Comment (matiu) [inline cordova/build.sh]: Maybe is better too use $OSTYPE
(like `if [[ "$OSTYPE" == "win32" ]]; then`)

Comment (cmgustavo) [inline cordova/build.sh]: I think that some platforms don't have $OSTYPE, so using $CURRENT_OS is more secure because it is passed from the command line

Comment (matiu) [inline cordova/build.sh]: ok, could you please add a 'echo "Using plugin XXXX for WINDOWS PHONE" so if windows phone release version 9 we can at least track the error?

**Comment by cmgustavo** [comment]
Updated.

**Pull request merged**